### ISSUE TITLE
Update to 2.5 [site.url] is not set

### ISSUE THREAD

**Issue by ncang** [danpros/htmly#142]
hello, i use free hosting and install HTMLy 2.4. This morning i see updater from admin panel, 
when i click & reload
<blockquote>[site.url] is not set</blockquote>
& blank

**Comment by Kanti** [comment]
hi @ncang,
sorry that I let you wait, have you solved the problem yet?

perhaps the .htaccess changed during the update process?

**Comment by stingray82** [comment]
I also have this issue, auto update or manual upload same error when i try and enter the admin area, i have rolled back to my previous version now until i can find a solution

**Comment by Raff80** [comment]
hmmm...I have the same problem after updating it

**Comment by ncang** [comment]
no problem @Kanti , i use another old release :+1:  like @stingray82 say

**Comment by Kanti** [comment]
hi @ncang , @stingray82 
now this bug is fixed in v2.5.1.
If you are on version 2.5 put this line in your index.php:
```php
$config_file = 'config/config.ini';
```
i should than look like this:
```php
<?php
$config_file = 'config/config.ini';

require 'system/vendor/autoload.php';
require 'system/htmly.php';
```
